causalml/inference/meta/rlearner.py

```diff
@@ -182,7 +182,8 @@ def fit_predict(self, X, p, treatment, y, return_ci=False,
 
             return (te, te_lower, te_upper)
 
-    def estimate_ate(self, X, p, treatment, y):
+    def estimate_ate(self, X, p, treatment, y, return_ci=False, n_bootstraps=1000, bootstrap_size=10000,
```

Comment:
Let's change `return_ci` to `bootstrap_ci` because `estimate_ate` will always return CIs.